product_security/recommendations_for_customers/restrict_access.adoc

```diff
@@ -0,0 +1,200 @@
+`test1234` project, run:
+
+----
+$ oc policy add-role-to-group edit system:serviceaccount:managers -n test1234
+----
+
+The system ensures that service accounts always have an API token and registry
+credentials. The generated API token and registry credentials do not expire, but
+they can be revoked by deleting the secret. When the secret is deleted, a new
+one is automatically generated to take its place.
+
+In addition to providing API credentials, a pod’s service account determines
+which secrets the pod is allowed to use. Pods use secrets in two ways:
+
+* Image pull secrets, providing credentials used to pull images for the pod’s
+containers.
+* Mountable secrets, injecting the contents of secrets into containers as files.
+
+To allow a secret to be used as an image pull secret by a service account’s
+pods, run:
+
+----
+$ oc secrets link --for=pull <serviceaccount-name> <secret-name>
+----
+
+To allow a secret to be mounted by a service account’s pods, run:
+
+----
+$ oc secrets link --for=mount <serviceaccount-name> <secret-name>
+----
+
+See
+xref:../../admin_guide/service_accounts.adoc#admin-managing-service-accounts[Configuring
+Service Accounts] for more information.
+
+[[security-service-accounts]]
+== OOTB Application-level Public Key Infrastructure
+
+{product-title} features an application-level public key infrastructure (PKI).
+This PKI manages certificates generated with the service serving certificate
+secret feature. The Root CA files for this PKI are stored on master nodes under
+`/etc/origin/master/service`.
+
+Service serving certificate secrets are intended to support complex middleware
+applications that need out-of-the-box certificates. To secure communication to
+your service, have the cluster generate a signed serving certificate/key pair
+into a secret in your namespace. The certificate and key are in Privacy-Enhanced
+Mail (PEM) format, stored in `tls.crt` and `tls.key` respectively. The
+certificate/key pair is automatically replaced when it gets close to expiration.
+The signature algorithm for this feature is `x509.SHA256WithRSA`. To manually
+rotate the certificate/key pair, delete the generated secret and a new
+certificate will be created. See
+xref:../../dev_guide/secrets.adoc#service-serving-certificate-secrets[Service
+Serving Certificate Secrets] for more information.
+
+[[dedicated-network-security-restricting-network-access]]
+== Restricting Network Access
+
+Traffic inbound to the cluster can be restricted at the platform level. If using
+a VPN connection or a VPC peer, access to the cluster can be restricted so that
+only internal traffic is accepted. Route-specific IP whitelisting can also be
+implemented.
+
+[[dedicated-network-security-routes]]
+=== Routes
+
+See the
+xref:../../architecture/networking/routes.adoc#architecture-core-concepts-routes[Routes]
+topic for an overview of routes, including route-specific annotations.
```

Comment:
add after including route-specific annotations - " and IP whitelisting"

Reply:
Adding in the text, but I'll leave it unbolded per our style guidelines